File: js/agingThePopulation.js
// Given an object of people and their ages, return how old the people would be after n years have passed.
// Use the absolute value of n.
// https://edabit.com/challenge/xtDkfWxEh2hbmAHvn

const afterNYears = (names, n) => {
  let updatedNames = {};

  for (const name in names) {
    Object.assign(updatedNames, { [name]: names[name] + Math.abs(n) });
  }

  return updatedNames;
};

module.exports = afterNYears;

File: python/aging_the_population.py
"""
Aging the Population

Given a dictionary of people and their ages, return how old the people would be after n years have passed. Use the absolute value of n.

https://edabit.com/challenge/zY8WSKGcqmkGd73Wr
"""
from typing import Dict


def after_n_years(names: Dict[str, int], n: int) -> Dict[str, int]:
    return {key: value + abs(n) for key, value in names.items()}
